plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -106,6 +132,22 @@ instance Pretty DefaultFun where
     pretty Nop1                 = "nop1"
     pretty Nop2                 = "nop2"
     pretty Nop3                 = "nop3"
+    pretty Fst                  = "fst"
+    pretty Snd                  = "snd"
+    pretty Null                 = "null"
+    pretty Head                 = "head"
+    pretty Tail                 = "tail"
+    pretty ConstrC              = "constrC"
+    pretty MapC                 = "mapC"
+    pretty ListC                = "listC"
+    pretty IC                   = "iC"
+    pretty BC                   = "bC"
+    pretty UnConstrC            = "unConstrC"
+    pretty UnMapC               = "unMapC"
+    pretty UnListC              = "unListC"
+    pretty UnIC                 = "unIC"
+    pretty UnBC                 = "unBC"
+    pretty ChooseData           = "chooseData"
```

Comment:
I wanted to do that in a separate PR and make this one minimal, so that people can check it for how to add a new complex built-in type, but since I've clearly failed at that, I'll just change the `pretty` stuff as well.